setup.py

```diff
@@ -83,7 +83,7 @@ def read(fname):
     entry_points={'console_scripts': [
         'django-admin = django.core.management:execute_from_command_line',
     ]},
-    install_requires=['pytz', 'sqlparse', 'asgiref'],
+    install_requires=['pytz', 'sqlparse', 'asgiref', 'aiofile'],
```

Comment:
We probably shouldn't include `aiofile` as a dependency since it is only used when the cache backend is set to filebased.

We can conditionally import it when needed as it's done for memcached: https://github.com/nicolaslara/django/blob/e8e6ea7123ca6ae170898d4ac170a1bd30c1c0f9/django/core/cache/backends/memcached.py#L181. We could return a nice error if it is missing.

Reply:
Sounds good, I will make that change!